src/slices/comments.slice.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { getPostComments } from '../api/comments';
+
+type Comments = {
+  loaded: boolean;
+  hasError: boolean;
+  items: Comment[];
+};
+
+const initialState: Comments = {
+  loaded: false,
+  hasError: false,
+  items: [],
+};
+
+export const fetchPostComments = createAsyncThunk<Comment[], number>(
+  'comments/fetchPostComments',
+  async (postId: number) => {
+    const result = await getPostComments(postId);
+
+    return result;
+  },
+);
+
+const commentsSlice = createSlice({
+  name: 'usersList',
```

Comment:
The slice name 'usersList' seems incorrect for a comments slice. Consider renaming it to something more appropriate like 'comments'.